src/sim/trace-extract/google_trace_simulator.cc

```diff
@@ -171,10 +311,46 @@ void GoogleTraceSimulator::Run() {
       LOG(ERROR) << "Could not open bin output file.";
     }
   } else {
-    ReplayTrace();
+    ofstream *stats_file = NULL;
+    if (!FLAGS_stats_file.empty()) {
+      stats_file = new ofstream(FLAGS_stats_file);
+      if (stats_file->fail()) {
+        LOG(FATAL) << "Could not open for writing stats file "
+                   << FLAGS_stats_file;
+      }
+    }
+
+    graph_output_ = NULL;
```

Comment:
Agree, though maybe we should make an issue for this and leave it as-is in this PR. Alternatively, re-engineer the code to make stats_file a FILE* everywhere. (Google style guide says that streams should only be used for logging.)